sormas-backend/src/main/java/de/symeda/sormas/backend/task/TaskFacadeEjb.java

```diff
@@ -914,19 +914,34 @@ public TaskDto getByUuid(String uuid) {
 
 	@Override
 	@RightsAllowed(UserRight._TASK_DELETE)
-	public void deleteTask(TaskDto taskDto) {
-		Task task = taskService.getByUuid(taskDto.getUuid());
+	public void delete(String uuid) {
+		if (!userService.hasRight(UserRight.TASK_DELETE)) {
```

Comment:
Why is this needed? Shouldn't the `@RightsAllowed` annotation already handle this?

Reply:
It was like this in the past ( on development): 
![image](https://github.com/SORMAS-Foundation/SORMAS-Project/assets/47103965/a1b0e6c9-ed30-46f5-809e-771a87d68792)

And when the user does not have the TASK_DELETE right the delete will still appear in the bulk dropdown.
The annotation helps the interceptor to catch the problem and will be visible in the server.log but the user will not be notified, the task will remain selected and nothing will happen after clicking the delete.  
![image](https://github.com/SORMAS-Foundation/SORMAS-Project/assets/47103965/9c1d612a-5f1f-4b79-a4f9-d87949b3f960)
![image](https://github.com/SORMAS-Foundation/SORMAS-Project/assets/47103965/724e86c1-2260-45e1-a2d5-7452e29b21ec)

With the added exception in the same scenario the user will be notified about what happened:
![image](https://github.com/SORMAS-Foundation/SORMAS-Project/assets/47103965/a3fc9a54-36ef-47d7-af2c-90e351a88139)


I saw that we solved like this the problem on short term in case of other methods too:
![image](https://github.com/SORMAS-Foundation/SORMAS-Project/assets/47103965/531ae371-85f2-4c49-a380-12e3706b0a7e)
![image](https://github.com/SORMAS-Foundation/SORMAS-Project/assets/47103965/3cf7c481-b5e7-4241-ba6e-1d179d06d873)

The long term solution would be to implement this ticket and do not display at all the button/ menu item when the user does not have the proper right:
https://github.com/SORMAS-Foundation/SORMAS-Project/issues/12015